android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -1908,7 +2025,24 @@ else if ( "O".equalsIgnoreCase( gender ) )
         return AppLovinGender.UNKNOWN;
     }
 
-    private static AppLovinAdContentRating getAppLovinAdContentRating(int maximumAdContentRating)
+    private static String getRawAppLovinGender(final AppLovinGender gender)
+    {
+        switch ( gender )
+        {
+            case UNKNOWN:
+                return "U";
+            case FEMALE:
+                return "F";
+            case MALE:
+                return "M";
+            case OTHER:
+                return "O";
+            default:
+                return "U";
```

Comment:
We just repeated `"U"` which goes against one of the top rules of AppLovin engineering - we repeated ourselves. Can you tell me why you want to use a `switch` statement over an `if` statement? `if` statement will be easier to use and you can just return `"U"` in the `else` case.

I can see the argument for `switch` statement for enums with many values, since compiler will provide warning if you missed a case.

Reply:
Has been fixed to use `if` statement.